aim/api/resource.py

```diff
@@ -86,7 +89,6 @@ class Tenant(AciResourceBase):
 
     identity_attributes = ['name']
     other_attributes = ['display_name']
-    db_attributes = []
 
     _aci_mo_name = 'fvTenant'
     _tree_parent = None
```

Comment:
I see now that first level objects don't have FK constraints... This could be an issue for the synchronization, but let's merge it for now and see if we can circle this problem around without having the Northbound project creating a tenant.